Documentation/Blazorise.Docs/Pages/Blog/2024-05-17_HowToCreateSocialMediaShareButtons/Index.md

```diff
@@ -0,0 +1,205 @@
+
+---
+
+## Let's dive in and jazz up your app with these fantastic buttons.
+
+## Installing Blazorise
+
+Run the following commands in your terminal to add Blazorise as a dependency to your Blazor app.
+
+```bash|InstallBlazorise
+Install-Package Blazorise.Bootstrap
+Install-Package Blazorise.Icons.FontAwesome
+```
+
+Alternatively you can use your favorite IDE's nuget package manager and add the packages automatically!
+
+---
+
+## Adding static files in `wwwroot/index.html`
+
+```html|HeadContent
+<html>
+<head>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" 
+        integrity="sha384-zCbKRCUGaJDkqS1kPbPd7TveP5iyJE0EjAuZQTgFLD2ylzuqKfdKlfG/eSrtxUkn" 
+        crossorigin="anonymous">
+
+  <link href="_content/Blazorise.Icons.FontAwesome/v6/css/all.min.css" rel="stylesheet">
+  <link href="_content/Blazorise/blazorise.css" rel="stylesheet" />
+  <link href="_content/Blazorise.Bootstrap/blazorise.bootstrap.css" rel="stylesheet" />
+</head>
+</html>
+```
+
+We can see that we are adding a few links to our head tag, let's break them down:
+- The first link, adds a link to BootstrapCSS, this is required in order for our buttons to work! as we are using Bootstrap for this tutorial.
+- The rest of the links, add Blazorise's dependencies to the web app. This is also very important!
+
+---
+
+## Adding Imports inside of the `_Imports.razor`
+
+```html|UsingStatement
+@using Blazorise
+```
+
+---
+
+## Registering services
+
+Add the following line at the top of your `Program.cs` file:
+
+```cs|ProgramCsUsingStatements
+using Blazorise;
+using Blazorise.Bootstrap;
+using Blazorise.Icons.FontAwesome;
+```
+
+and somewhere inside the file, register Blazorise's services like so:
+
+```cs|ServiceRegistration
+builder.Services
+    .AddBlazorise()
+    .AddBootstrapProviders()
+    .AddFontAwesomeIcons();
+```
+
+---
+
+## Creating the ShareButton component
+
+```html|ShareButtonComponentMarkup
+<Button TextColor="@TextColor" Background="@(new Background(BackgroundColor))" To="@To" 
+        Type="@ButtonType.Link" Size="Size.Large" @attributes="@AdditionalAttributes">
+
+  @ChildContent
+
+  <Icon Name="@IconName" IconStyle="IconStyle.Light"/>
+</Button>
+```
+```cs|ShareButtonComponentCode
+@code 
+{
+  [Parameter]
+  public string TextColor { get; set; }
+
+  [Parameter]
+  public string BackgroundColor { get; set; }
+
+  [Parameter]
+  public string IconName { get; set; }
+
+  [Parameter, EditorRequired]
+  public string To { get; set; }
+
+  [Parameter, EditorRequired]
+  public RenderFragment ChildContent { get; set; }
+
+  [Parameter(CaptureUnmatchedValues = true)]
+  public Dictionary<string, object> AdditionalAttributes { get; set; } = [];
+}
+```
+
+## Let's break down the component
+
+### Button
```

Comment:
When breaking it down, maybe add a number in front of the title. eg, 1. Button, 2. The Parameters, etc.